web-common/src/Editor.purs

```diff
@@ -1,30 +1,27 @@
 module Editor
   ( EditorAction(..)
   , editorPane
-  , demoScriptsPane
   , withEditor
   ) where
 
 import Ace.EditSession as Session
 import Ace.Editor as Editor
 import Ace.Halogen.Component (AceMessage, AceQuery(..), Autocomplete(Live), aceComponent)
 import Ace.Types (Editor)
-import AjaxUtils (ajaxErrorPane)
-import Bootstrap (btn, btnDanger, btnInfo, btnPrimary, btnSecondary, btnSmall, btnSuccess, empty, listGroupItem_, listGroup_, pullRight)
+import AjaxUtils (ajaxErrorPane) -- Problem
```

Comment:
?

Reply:
Oops, that shouldn't have been committed. Will squash. Thanks.